### DISCUSSION TITLE
Filament Sensor

### DISCUSSION THREAD

**Discussion by thirdofive** [Jyers/Marlin#1923] [Q&A]
Does the firmware for the bltouch work with a filament sensor. I have tried 3 different filament sensors and and non of them work. Running V4.2.7 and E3V2-BLTouch-3x3-HS-v4.2.7-v2.0.1.bin I do not have the expertise to compile anything, I just want a filament sensor that will work. Any input would be appreciated.

**Comment by zoomming** [comment]
Hi ! Did you check/select the filament sensor box in the configuration menu ? By default it is unchecked...

**Reply by thirdofive** [reply]
Yes, I did check it, either way, it says there is no filament. Just wondering if anyone else has it working with this firmware, not stock firmware. If so, what sensor did you use?

**Reply by PapsDizzy** [reply]
> Yes, I did check it, either way, it says there is no filament. Just wondering if anyone else has it working with this firmware, not stock firmware. If so, what sensor did you use?

i use a cheap 1 euro sensor from aliexpress and its working
